README.md

```diff
@@ -43,11 +43,25 @@ export default {
         {find:'somelibrary-1.0.0', replacement: './mylocallibrary-1.5.0'}, //remap a library with a specific version
         {find:/^i18n\!(.*)/, replacement: '$1.js'}, //remove something in front of the import and append an extension (e.g. loaders, for files that were previously transpiled via the AMD module, to properly handle them in rollup as internals now)
         //for whatever reason, replace all .js extensions with .wasm
-        {find:/^(.*)\.js$/, replacement: '$1.wasm'} 
+        {find:/^(.*)\.js$/, replacement: '$1.wasm'}
       ]
     })
   ],
 };
+
+// or with object syntax
+export default {
+  input: './src/index.js',
+  plugins: [
+    alias({
+      resolve: ['.jsx', '.js'],
+      entries: {
+        'something': '../../../something',
```

Comment:
```suggestion
        something: '../../../something',
```

Props that don't need to be quoted typically shouldn't be.

Reply:
yeah, I've just moved and formatted the array format and didnt remove it here